code/modules/admin/verbs/randomverbs.dm

```diff
@@ -991,3 +991,63 @@ var/list/datum/outfit/custom_outfits = list() //Admin created outfits
 
 	for(var/obj/machinery/shuttle_manipulator/M in machines)
 		M.ui_interact(usr)
+
+/client/proc/mass_zombie_infection()
+	set category = "Fun"
+	set name = "Mass Zombie Infection"
+	set desc = "Infects all humans with a latent organ that will zombify \
+		them on death."
+
+	if(!holder)
+		return
+
+	var/confirm = alert(src, "Please confirm you want to add latent zombie organs in all humans?", "Confirm Zombies", "Yes", "No")
+	if(confirm != "Yes")
+		return
+
+	for(var/mob/living/carbon/human/H in mob_list)
+		new /obj/item/organ/body_egg/zombie_infection(H)
+
+	message_admins("[key_name_admin(usr)] added a latent zombie infection to all humans.")
+	log_admin("[key_name(usr)] added a latent zombie infection to all humans.")
+	feedback_add_details("admin_verb","MZI")
+
+/client/proc/polymorph_all()
+	set category = "Fun"
+	set name = "Polymorph All"
+	set desc = "Applies the effects of the bolt of change to every single mob."
+
+	if(!holder)
+		return
+
+	var/confirm = alert(src, "Please confirm you want polymorph all mobs?", "Confirm Polymorph", "Yes", "No")
+	if(confirm != "Yes")
+		return
+
+	var/keep_name = alert(src, "Do you want mobs to retain their names?", "Keep The Names?", "Yes", "No")
+
+	var/list/mobs = shuffle(living_mob_list.Copy()) // might change while iterating
+	var/who_did_it = key_name_admin(usr)
+	spawn(0)

```

Comment:
HELL NO WE WON'T GO

just make it spawn a short amount so other things can clear this tick.